linux-tkg-config/6.5/minimal-modprobed.db

```diff
@@ -25,9 +25,9 @@ algif_hash
 algif_skcipher
 alx
 amd64_edac
-amdgpu
```

Comment:
Why  is the order changing? Are you putting this list in `~/.config/modprobed.db` then calling `modprobed-db store` to get its own ordering ?

Reply:
I looked at the modprobed-db script and checked how it sorts, then applied that to the file. It uses `sort -k 1,1` to sort the file.

Reply:
I just did the procedure I suggested on `minimal-modprobed.db` of this PR and the order does not get changed with respect to master.

[minimal-modprobed.db.txt](https://github.com/Frogging-Family/linux-tkg/files/12840986/minimal-modprobed.db.txt)

So probably the tool `modprobed-db` is doing something more. If you can figure out exactly how it's ordered would be nice yeah. Otherwise just run modprobed-db on it and we're done x)

Reply:
I've just run `modprobed-db store` on the `minimal-modprobed.db` from master, and it changes the ordering to the one in this PR, so that combined with reading the code makes me quite sure that `sort -k 1,1` is the correct ordering.

Reply:
Ah that's interesting, I have no idea why my machine gives the other order. `sort` comes from the `coreutils` package so it doesn't depend on the shell :thinking:

Reply:
It might be that it doesn't sort it if there are no modules to be added?